### ISSUE TITLE
Emmiters ignore all laws of physics

### ISSUE THREAD

**Issue by Metichi** [Apollo-Community/ApolloStation#960]
So far I have found that:
- An emmiter will turn on when not connected to anything
- An emmiter's beam will DESTrOY walls (at least not reinforced ones)
- If on and displaced, the emmiter beam will stay in place while the emmiter is still on

Still need a little bit of testing but so far it looks like that

Messing with this will accumulate lag and collapse the sever

**Comment by halworsen** [comment]
2/3 of these are rooted in an exploit which isn't directly related to emitters.

**Comment by Kwask** [comment]
bumping this up to urgent priority because it can crash the server

**Comment by rjtwins** [comment]
So... was this actually fixed ?

**Comment by halworsen** [comment]
No, because two of the points seem impossible to me. I've tested locally, and the emitter will not turn on without power. This isn't something I changed when we switched to continuous beams. I've also tried moving the emitter while it's on, which has never worked either, because the emitter is anchored to the floor.

You told me how you moved the emitter (which I won't reveal here for obvious reasons), but I tried that too and nope. I don't understand how you've managed to get to the point where the emitters turn into server-crashing beasts.

Also I think I nerfed emitter damage against walls.
